tests/test_docs/test_code_blocks.py

```diff
@@ -148,6 +148,10 @@ class TestYamlSnippets(BaseTestDocCode):
         "docs/guides/running_service_different_networks.md": {"skip_blocks": [0]},
     }
 
+    skipped_files = [
+        "docs/guides/service_level_overrides.md",
```

Comment:
why?

Reply:
Because the configs defined here are dummy configs and does not have any actual config files to compare the code blocks against. 

https://github.com/valory-xyz/open-autonomy/blob/678f52220ad697cb8fe9ab82aff4aaa424207f9d/tests/test_docs/test_code_blocks.py#L133-L144